app/view_models/CapitalGains.scala

```diff
@@ -41,7 +41,7 @@ case class CapitalGains(
   forename: String,
   surname: String)
     extends GenericViewModel {
-
+  def isPaye = utr.isEmpty
```

Comment:
No isPaye needed for CapitalGains. This section is only shown for Self Assessment.

Reply:
Ok, I will remove it.